aloha-core/src/main/scala/com/eharmony/aloha/models/multilabel/package.scala

```diff
@@ -0,0 +1,74 @@
+package com.eharmony.aloha.models
+
+import com.eharmony.aloha.dataset.density.Sparse
+
+import scala.collection.{immutable => sci}
+
+/**
+  * Created by ryan.deak on 8/31/17.
+  */
+package object multilabel {
+
+  // All but the last type are package private, for testing.  The last is public.
+
+  /**
+    * Features about the input value (NOT including features based on labels).
```

Comment:
I agree with this, how hard is this to migrate?

Reply:
The `sci.IndexedSeq`?  It's not efficient enough when I tested a long time ago.